tests/unit/CODDate.spec.ts

```diff
@@ -109,42 +109,75 @@ describe('COD Date - COOPs', () => {
     expect(valids[1]).toEqual([true])
   })
 
-  it('invalidates the component when entered month is after Max Date', () => {
+  it('invalidates the component when entered date is after Max Date', () => {
     wrapper = mount(CodDate, {
       store,
       vuetify,
       computed: {
         minDate () {
-          return '2019-01-01'
+          return '2019-05-05'
         },
         maxDate () {
-          return '2019-05-05'
+          return '2019-05-10'
+        },
+        codDateRules () {
+          return []
         }
       }
     })
 
-    wrapper.setData({ dateFormatted: '2019/11/11' })
-    wrapper.vm.$v.$touch()
```

Comment:
In theory this should have triggered validation, but it didn't work in this case.

This test (and the one below, actually), failed to check `isNotNull` first, so the actual error message here was "A Director change date is required." ... clearly not what we are supposed to be testing.

Fixed now. (I check the actual message.)